labs-ios-starter/Model/Profile.swift

```diff
@@ -10,9 +10,9 @@ import UIKit
 
 struct Profile: Codable {
     
-    let id: String
-    let email: String
-    let name: String
+    let id: String?
+    let email: String?
+    let name: String?
     let avatarURL: URL?
     
```

Comment:
Good stuff! Make sure we dont return any errors in case any of these variables returns back nil!

Reply:
Johnny asked us to do this for the profile model